integration-tests/README.md

```diff
@@ -0,0 +1,107 @@
+| RESULT_ROOT  | Root directory for the integration test results and artifacts generated during test execution. | /tmp/it-testsresults
+| BASE_IMAGES_REPO  | The repository URL to download WebLogic and FMW image. Make sure you have access to this repository. Other supported repository is phx.ocir.io (OCIR)  | container-registry.oracle.com(OCR)
+| WEBLOGIC_IMAGE_NAME  | Name of the WebLogic image in the chosen repository.| middleware/weblogic (OCR) 
+| WEBLOGIC_IMAGE_TAG  | The tag for WebLogic base image. Generally, it represents the WebLogic version with JDK and/or installation type. Possible values are 12.2.1.3, 12.2.1.3-dev, 12.2.1.4-slim, 14.1.1.0-11 or 14.1.1.0-8. Please check the repository for the availability of these images. | 12.2.1.4
+| FMWINFRA_IMAGE_TAG  | The tag for Fusion Middleware Infrastructure base image. Generally, it represents the FMW version. Possible values are 12.2.1.3, 12.2.1.4. Please check the repository for the availability of these images. | 12.2.1.4
+| FMWINFRA_IMAGE_NAME  | Name of the Fusion Middleware Infrastructure image in the chosen repository.| middleware/fmw-infrastructure (OCR) 
+
+## Logging/Archiving
+
+On completion of integration test execution, the results are archived in a directory based on environment variable  LOG_DIR  (the default value is /tmp/it-testsresults/it-diagnostic) and runtime artifacts are available in a directory based on environment variable RESULTS_ROOT (the default value is /tmp/it-testresults) . 
+
+A typical diagnosticlogs directory content will look as follows after the completion of test ItMiiDomain. The standard output for the tests is captured in ItMiiDomain.out. For each test method ( say testCreateMiiDomain, testCreateMiiSecondDomain) a directory is created and the corresponding k8s object description log(s) and server pod logs are saved if the test fails or environment variable COLLECT_LOGS_ON_SUCCESS set to true.
+
+```
+itMiiDomain
+
+|-- ItMiiDomain.out
+
+|-- testCreateMiiDomain
+
+|   |-- ns-gggs.list.services.log
+
+|   |-- ns-gggs.pod.weblogic-operator-7b8-lwqzm.container.weblogic-operator.log
+
+|   |-- ns-luaf.pod.domain1-admin-server.container.weblogic-server.log
+
+|   |-- ns-luaf.pod.domain1-managed-server1.container.weblogic-server.log
+
+|   |-- ns-luaf.pod.domain1-managed-server2.container.weblogic-server.log
+
+-- testCreateMiiSecondDomain
+
+    |-- ns-gggs.list.configmaps.log
+
+    |-- ns-gggs.list.domains.log
+
+    |-- ns-gggs.list.services.log
+
+    |-- ns-gggs.pod.weblogic-operator-7b8f-lwqzm.container.weblogic-operator.log
+
+    |-- ns-luaf.list.configmaps.log
+
+    |-- ns-luaf.pod.domain1-admin-server.container.weblogic-server.log
+
+    |-- ns-luaf.pod.domain1-managed-server1.container.weblogic-server.log
+```
+## Troubleshooting
+The diagnostic logs files in method scoped directory in ${LOGS_DIR}/`<testClass>`/`<testMehod>` and the test stdout ${LOGS_DIR}/`<testClass>`.out is the starting point to triage a test failure.
```

Comment:
in method scoped directory -> in the method-scoped directory

Reply:
done